### PR TITLE
Fixed incorrect deployment function name causing APM + logging to be …

### PR DESCRIPTION
**Pull request by johanfol**
…wrong;

Fixed issue where rule results are being sent to typology processor multiple times.

### CODE REVIEW

**Review by vorsterk — APPROVED** [review]
Looks good!

File: src/controllers/execute.ts [hunk 1]
```diff
@@ -57,10 +57,16 @@ const sendRuleResult = async (ruleResult: RuleResult, req: RuleRequest) => {
...
     for (const typology of channel.typologies) {
       if (typology.rules.some((rule) => rule.id === ruleResult.id)) {
-        const typologyResponse = await axios.post(`${typology.host}/execute`, toSend);
+        const typologyResponse = await axios.post(
+          `${typology.host}/execute`,
+          toSend
+        );
         if (typologyResponse.status !== 200) {
-          LoggerService.error(typologyResponse.data);
+          LoggerService.error(
```
Comment (uche-lextego) [inline src/controllers/execute.ts:65]: @johanfol you handled it from TP and you are logging the errors on the rule processor???

Comment (johanfol) [reply src/controllers/execute.ts:65]: Only logging error from TP IF there was a non-200 status code, indicative of an error. Furthermore, the "return" below causes that we will not submit the request to the TP multiple times. Lastly, the TP don't throw an error if the same rule result is submitted multiple times, that would be too disruptive, so I only handle the duplication by returning a 200 success message to the Rule Processor before any processing get's done.

Comment (uche-lextego) [reply src/controllers/execute.ts:65]: okay, I understand now.

**Review by uche-lextego — APPROVED** [review]

**Pull request merged**